### PR TITLE
Remove deprecated code in 0.19

### LINKED ISSUE #10746: Remove code deprecated in 0.19 (released on December 06, 2021)
There is code released 20 month ago that needs to be removed.
 - `qiskit/transpiler/instruction_durations.py:173` (`get`)
 - `qiskit/providers/models/backendconfiguration.py:834` (`control`)
 - `qiskit/qobj/__init__.py:84` (`__init__`)
  
This includes:
- [ ] Remove deprecated code and tests (if they only check the deprecation raise)
- [ ] tag the PR as [`Changelog: Removal`](https://github.com/Qiskit/qiskit/labels/Changelog%3A%20Removal)
- [ ] create a release note in the category `upgrade:`. If you can include an example with an alternative for user to migrate to the new code, as this change might break users code.

### PR DESCRIPTION
**Pull request by doublehoon**
### Summary
Remove deprecated code in 0.19

### Details and comments
Fixes #10746 
Please add `Changelog: Removal` label.

### CODE REVIEW

File: qiskit/qobj/__init__.py
Comment (Raghav-Bell) [inline qiskit/qobj/__init__.py:1]: `docs test` are failing due to `Qobj` please remove this.

**Review by Raghav-Bell — CHANGES_REQUESTED** [review]
`docs tests` are failing due to `Qobj`  in `docstring` even after removal, please remove it from  below line.
https://github.com/Qiskit/qiskit/blob/cd507adc68a96a333bc62de7ce9799a245506f25/qiskit/qobj/__init__.py#L26

File: qiskit/transpiler/instruction_durations.py [hunk 1]
```diff
@@ -170,19 +169,10 @@ def update(self, inst_durations: "InstructionDurationsType" | None, dt: float =
...
         return self
 
-    @deprecate_arg(
-        "qubits",
-        deprecation_description=(
-            "Using a Qubit or List[Qubit] for the ``qubits`` argument to InstructionDurations.get()"
-        ),
-        additional_msg="Instead, use an integer for the qubit index.",
-        since="0.19.0",
-        predicate=_is_deprecated_qubits_argument,
```
Comment (1ucian0) [inline qiskit/transpiler/instruction_durations.py:180]: I think `_is_deprecated_qubits_argument` (def in line 27) can also be removed, if this is the only usage.

File: releasenotes/notes/remove-deprecated-code-in-0.19-a97ccfec62405b9a.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    In the internal ``qiskit.transpiler.instruction_durations.get`` object, using a Qubit or List[Qubit] for the argument ``qubits`` has been deprecated. Instead, use an integer for the qubit indices.
```
Comment (1ucian0) [inline releasenotes/notes/remove-deprecated-code-in-0.19-a97ccfec62405b9a.yaml]:
```suggestion
    The argument ``qubits`` in the method :meth:`qiskit.transpiler.instruction_durations.InstructionDurations.get`, does not accept :class:`.Qubit` (or a list of them) any more. This functionality was deprecated in Qiskit 0.33 (with Terra 0.19), released on Dec 2021. Instead, use an integer for the qubit indices.
```

File: releasenotes/notes/remove-deprecated-code-in-0.19-a97ccfec62405b9a.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+  - |
+    In the internal ``qiskit.providers.models.backendconfiguration.control`` object, the argument ``channel`` has been deprecated. Instead, use the ``qubits`` argument.
```
Comment (1ucian0) [inline releasenotes/notes/remove-deprecated-code-in-0.19-a97ccfec62405b9a.yaml]:
```suggestion
    The argument ``channel`` in the method :meth:`qiskit.providers.models.backendconfiguration.PulseBackendConfiguration.control` is removed. It was deprecated in Qiskit 0.33 (with Terra 0.19), released on Dec 2021. Instead use  the ``qubits`` argument.
```

File: releasenotes/notes/remove-deprecated-code-in-0.19-a97ccfec62405b9a.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+  - |
+    In the internal ``qiskit.qobj.__init__`` object, the initializer function ``__init__`` has been deprecated. Instead, use the ``QasmQobj`` or ``PulseQobj`` object.
```
Comment (1ucian0) [inline releasenotes/notes/remove-deprecated-code-in-0.19-a97ccfec62405b9a.yaml]:
```suggestion
    The class ``qiskit.qobj.Qobj`` is removed. It was deprecated in Qiskit 0.33 (with Terra 0.19), released on Dec 2021. Instead, use :class:`qiskit.qobj.QasmQobj` or :class:`qiskit.qobj.PulseQobj`.
```

**Review by 1ucian0 — COMMENTED** [review]
Great work @DoubleHoon! some small comments on the reno.

(and thanks @Raghav-Bell for the review)

File: qiskit/transpiler/instruction_durations.py (hunk 1 above)
Comment (doublehoon) [reply qiskit/transpiler/instruction_durations.py:180]: Thanks! Removed `_is_deprecated_qubits_argument` as you suggested.

**Review by 1ucian0 — APPROVED** [review]
Really good work. Welcome to the project!

**Pull request merged**